docs/install.md

```diff
@@ -529,21 +529,22 @@ The following is the procedure to globally configure the username used by SAT an
     Not generating configuration file.
     ```
 
-1.  Edit `~/.config/sat/sat.toml` and set the username option in the `api_gateway` section of the config file. E.g.:
+1.  Edit `~/.config/sat/sat.toml` and set the username option in the `api_gateway` section of the config file. For
+    example:
 
     ```screen
     username = "crayadmin"
     ```
 
-1.  Run `sat auth`. Enter your password when prompted. E.g.:
+1.  Run `sat auth`. Enter your password when prompted.
```

Comment:
Still feels like a "For example:" makes sense given that the username is an example.